### DISCUSSION TITLE
How to stop mouse events of Sprites from having their propagation stopped?

### DISCUSSION THREAD

**Discussion by ftzi** [pixijs/pixijs#9070] [Q&A] [answered]
I have some sprites that overlap. Only the one above is catching the mouse events and I am not calling e.stopPropagation().

Is there someway to propagate the event to all of those Sprites that are under the pointer?

**Comment by lykl** [comment]
sorry,I misunderstood.
```
const sprite1 = new Sprite()
const sprite2 = new Sprite()
[sprite1,sprite2].forEach(sprite=>{
    sprite.containsPoint(event.global) // it will return a boolean, if true, this point is inside the sprite
})
```
Maybe you can try like this.

**Reply by ftzi** [reply]
~~That doesn't work~~

**Comment by bigtimebuddy** [comment, marked as answer]
This works just like the DOM:
https://jsfiddle.net/bigtimebuddy/pk3uf96e/ (DOM)
https://jsfiddle.net/bigtimebuddy/4gujwxf6/ (PixiJS)

If you want to get all items under the point, you can try something like this:
https://jsfiddle.net/bigtimebuddy/d8sm0vc9/
